src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
```

Comment:
The width and height of elements should not be fixed. It's better to use min-width or max-width, and padding or margin for spacing to make the layout more responsive and flexible. Consider removing the fixed width and height and using other CSS properties for layout adjustments.